cfe.py

```diff
@@ -24,35 +83,26 @@ def run_cfe(self, cfe_state):
             self.et_from_rainfall(cfe_state)
         
         cfe_state.vol_et_from_rain += cfe_state.actual_et_from_rain_m_per_timestep
-        #cfe_state.vol_et_to_atm += cfe_state.actual_et_from_rain_m_per_timestep
+        cfe_state.vol_et_to_atm += cfe_state.actual_et_from_rain_m_per_timestep
         cfe_state.volout += cfe_state.actual_et_from_rain_m_per_timestep
 
-        # ________________________________________________
-        # Calculate evaporation from soil
```

Comment:
We need to include options for all the old code. We have to be able to run an identical version as the C code. So there needs to be an option from the configuration file to run this.